features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesPresenter.kt

```diff
@@ -176,8 +208,21 @@ class MessagesPresenter @AssistedInject constructor(
             .onFailure { Timber.e(it) }
     }
 
-    private fun notImplementedYet() {
-        Timber.v("NotImplementedYet")
+    private fun CoroutineScope.reinviteOtherUser(inviteProgress: MutableState<Async<Unit>>) = launch(dispatchers.io) {
+        suspend {
+            room.updateMembers()
+
+            val memberList = when (val memberState = room.membersStateFlow.value) {
+                is MatrixRoomMembersState.Ready -> memberState.roomMembers
+                is MatrixRoomMembersState.Error -> memberState.prevRoomMembers.orEmpty()
+                else -> emptyList()
+            }
+
+            val member = memberList.first { it.userId != room.sessionId }
```

Comment:
I was wondering if this could happen in a room that was just created, but I think `room.isDirect` would be false in that case and this point would never be reached.